processor/src/operations/io_ops.rs

```diff
@@ -223,9 +276,75 @@ mod tests {
         assert_eq!(1, process.memory.size());
         assert_eq!(word, process.memory.get_value(1).unwrap());
 
-        // --- calling STOREW with a stack of minimum depth is ok ----------------
+        // --- calling LOADW with a stack of minimum depth is ok ----------------
+        let mut process = Process::new_dummy();
+        assert!(process.execute_op(Operation::MLoadW).is_ok());
+    }
+
+    #[test]
+    fn op_mload() {
+        let mut process = Process::new_dummy();
+        assert_eq!(0, process.memory.size());
+
+        // push a word onto the stack and save it at address 2
+        let word = [1, 3, 5, 7].to_elements().try_into().unwrap();
+        store_value(&mut process, 2, word);
+
+        // push the address onto the stack and load the element
+        process.execute_op(Operation::Push(Felt::new(2))).unwrap();
+        process.execute_op(Operation::MLoad).unwrap();
+
+        let expected_stack = build_expected_stack(&[1, 7, 5, 3, 1]);
+        assert_eq!(expected_stack, process.stack.trace_state());
+
+        // check memory state
+        assert_eq!(1, process.memory.size());
+        assert_eq!(word, process.memory.get_value(2).unwrap());
+
+        // --- calling MLOAD with a stack of minimum depth is ok ----------------
+        let mut process = Process::new_dummy();
+        assert!(process.execute_op(Operation::MLoad).is_ok());
+    }
+
+    #[test]
+    fn op_mstore() {
+        let mut process = Process::new_dummy();
+        assert_eq!(0, process.memory.size());
+
+        // push new element onto the stack and save it as first element of the word on
+        // uninitialized memory at address 0
+        let element = Felt::new(10);
+        store_element(&mut process, 0, element);
+
+        // check stack state
+        let expected_stack = build_expected_stack(&[10]);
+        assert_eq!(expected_stack, process.stack.trace_state());
+
+        // check memory state
+        let mem_0 = [Felt::new(10), Felt::new(0), Felt::new(0), Felt::new(0)];
```

Comment:
You can probably simplify this as:
```Rust
let mem_0 = [element, ZERO, ZERO, ZERO];
```
You'll need to import `ZERO` from the core crate for this to work.